microsoftgraph/microsoft-graph-explorer-v4
# Fix: improve themes MessageBar colors

## Overview

MessageBars in dark theme and high contrast theme did not reflect theme colors. Semantic colors have now been added to both themes. Dark mode blue links are now also in line with the light blue in Fluent UI standards. 

### Demo

![image](https://user-images.githubusercontent.com/47487758/124667197-eaf53880-de63-11eb-850e-3c4e6f49020d.png)
![image](https://user-images.githubusercontent.com/47487758/124667314-15df8c80-de64-11eb-8932-1d05c2baf30d.png)

### Notes

Followed standards from documentation: 
https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar
![image](https://user-images.githubusercontent.com/47487758/124667256-03655300-de64-11eb-86b6-e72c4ab29d5d.png)

The semantic colors are from https://docs.microsoft.com/en-us/javascript/api/theme/isemanticcolors?view=office-ui-fabric-react-latest
I also looked into the source code for MessageBar: https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/MessageBar/MessageBar.styles.ts


## Testing Instructions

* Check out the branch, go to More Actions button to change theme. Notice MessageBars now reflect theme colors. 

https://dev.azure.com/microsoftgarage/Intern%20GitHub/_sprints/taskboard/GI21%20-%20Graph%20Explorer/Intern%20GitHub/Y21-S/04?workitem=38276
